### DISCUSSION TITLE
加入伟大的爱莉神教吧！

### DISCUSSION THREAD

**Discussion by EDEN54418** [share121/inter-knot#2688] [其他]
这里有你想要的一切
这里是梦想的天堂
爱莉希雅站领世界
![QQ图片20240818151836](https://github.com/user-attachments/assets/b891bb28-f8f0-4aab-a774-c9e12af3206f)

**Comment by dnmdngcsb** [comment]
支持！爱门！

**Comment by LBK-miq** [comment]
爱门！

**Comment by AnkaAlioth** [comment]
<font color=#e982c6>爱门!</font>
![aimen](https://github.com/user-attachments/assets/880e39c6-7db7-444c-b7ed-8fbea27f5018)

**Comment by wrxwyj** [comment]
爱门！

**Comment by AurEian** [comment]
爱门!!!
![previewFix](https://github.com/user-attachments/assets/fc7d8609-80d3-4f44-9474-6fa267213b56)

**Comment by qtsa66** [comment]
![Uploading 97810572_p0.jpg…]()
爱门！！

**Comment by shan-dgha** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" > 爱门

**Comment by OttoApocalypse01** [comment]
爱门

**Comment by Robei1** [comment]
爱门！

**Comment by yangtiansa** [comment]
✞爱门✞

**Comment by flamingo0106** [comment]
爱门♥

**Comment by Bailinghai** [comment]
爱门！<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >

**Comment by CJHXAMSTLP** [comment]
爱门

**Comment by shengwang999** [comment]
爱门！！！
![94c8d953d3cc97128d09f7d0c30e4d6](https://github.com/user-attachments/assets/387d5ba2-e165-4a33-a96f-ebb46ea54695)

**Comment by csy20070709** [comment]
爱门！！！

**Comment by CPKchenxi** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" ><img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >，

**Comment by wall-leg-sao-gang** [comment]
爱门！

**Comment by MOSTIMAPAN** [comment]
爱门！

**Comment by youyiMC** [comment]
![c3f2411db1e4ea1a](https://github.com/user-attachments/assets/a4627efc-73fd-42c1-abd3-1135b674e230)

**Comment by Kiiro-Elysia** [comment]
爱门

**Comment by 502awa** [comment]
爱门！

**Comment by cleo66** [comment]
爱门！

**Comment by SeeieKianna** [comment]
爱门!

**Comment by wuyueji** [comment]
爱门！

**Comment by 1ctrl-cv** [comment]
爱门！

**Comment by wzb768** [comment]
爱门

**Comment by wzb768** [comment]
爱门

**Comment by Hod-141** [comment]
爱门！<img src="https://upload-bbs.miyoushe.com/upload/2023/01/21/860c72d5521a0647514c324d16c7db1b_8005402887548479862.png" alt="爱莉希雅_安慰" width="75" >

**Comment by waler-ray** [comment]
爱门！

**Comment by G21674** [comment]
爱门！
